### ISSUE TITLE
[feature request] limit replacements per page

### ISSUE THREAD

**Issue by SGillessen** [CalderaWP/Caldera-Affiliates#3]
provide option to limit the replacement to x times per page, and scatter replacement randomly

Ex:

website contains xyc 10 times and I want it to be replaced 3 times with -link to xyz-, do the following:

xyc
-link to xyz-
xyc
xyc
xyc
-link to xyz-
xyc
-link to xyz-
xyc
xyc

**Comment by Shelob9** [comment]
Good idea. I was only thinking about limiting it to first occurrence, not the random.

**Comment by SGillessen** [comment]
@Shelob9 I added that because I had about 30 replacements on a single site on one product review ;)
